t/t2800-jobs-cmd.t

```diff
@@ -102,6 +102,14 @@ test_expect_success HAVE_JQ 'submit jobs for job list testing' '
 	echo $jobid >> inactiveids &&
 	echo $jobid > failed.ids &&
 	#
+	#  Run a job that will timeout, copy its JOBID to both inactive and
+	#   timeout lists.
+	#
+	jobid=`flux mini submit --time-limit=0.5s sleep 30` &&
+	echo $jobid >> inactiveids &&
+	echo $jobid > timeout.ids &&
+	fj_wait_event `tail -n 1 inactiveids` clean &&
```

Comment:
Not a big deal, but why get the jobid using `tail` when it is already in `$jobid`? (Assuming I missed something obvious, or this was just cut-and-paste from other tests)

Reply:
oh haha, it's cut and paste from the other tests.